main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"runtime"
+)
+
+// #include <unistd.h>
```

Comment:
This is most convenient way to get total memory turns out

https://stackoverflow.com/questions/30021296/how-to-get-total-memory-ram-in-go